go/mysql/flavor_mariadb.go

```diff
@@ -154,6 +155,31 @@ func (mariadbFlavor) status(c *Conn) (SlaveStatus, error) {
 	if err != nil {
 		return SlaveStatus{}, vterrors.Wrapf(err, "SlaveStatus can't parse MariaDB GTID (Gtid_Slave_Pos: %#v)", resultMap["Gtid_Slave_Pos"])
 	}
+	filePos, err := strconv.Atoi(resultMap["Exec_Master_Log_Pos"])
+	if err != nil && resultMap["Exec_Master_Log_Pos"] != "" {
+		return SlaveStatus{}, fmt.Errorf("invalid FilePos GTID (%v): expecting pos to be an integer", resultMap["Exec_Master_Log_Pos"])
+	}
+	file := resultMap["Exec_Master_Log_Pos"]
+	if file != "" {
```

Comment:
I was thinking we'd save the original string form of filePos as a var so we can leave the GTIDSet nil if either the file or pos was empty. Currently, we're treating an empty string pos the same as 0, but I don't think we should assume that.

Reply:
Done.